pages/events.jsx

```diff
@@ -120,7 +120,9 @@ var MakerPartyExample = React.createClass({
                 <span className="span-content label-tag">Participants</span><span className="span-content">{this.props.participants}</span>
               </div> : null
             }
-            <div className="description">{this.props.description}</div>
+            <div>
+              <p className="description" dangerouslySetInnerHTML={{__html: this.props.description}}></p>
+            </div>
```

Comment:
I like this suggestion and yay we should start this naming convention from here! :smile: